analyzers/its/expected/Nancy/Nancy--net452-S3900.json

```diff
@@ -2867,19 +2867,6 @@
 "uri":  "sources\Nancy\src\Nancy\Routing\Constraints\ParameterizedRouteSegmentConstraintBase.cs",
 "region":  {
 "startLine":  21,
-"startColumn":  89,
```

Comment:
Bunch of ITs have changed, with no change in UTs.
We need to document those cases (one per unique reason)